.devcontainer/devcontainer.json

```diff
@@ -67,8 +67,15 @@
         "config": "cp .devcontainer/.env.devcontainer .env",
         "apache": "sudo sed -i 's/Listen 80$//' /etc/apache2/ports.conf && sudo cp .devcontainer/apache-vhost.conf /etc/apache2/sites-enabled/000-default.conf && sudo a2enmod rewrite",
         "symfony": "wget https://get.symfony.com/cli/installer -O - | bash && sudo mv ~/.symfony5/bin/symfony /usr/local/bin/symfony"
-    }
+    },
```

Comment:
This will make it **invalid** JSON now with the rest commented out below. Please remove the `,` again here